<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculadora Álgebra LIneal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_calcular">
      <property name="text">
       <string>Calcular</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QComboBox" name="comboBox_calculator_mode">
      <item>
       <property name="text">
        <string>--Select--</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Normal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Números Complejos</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sistema de Ecuaciones (Sustitución)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sistema de Ecuaciones (Igualación)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sistema de Ecuaciones (Reducción)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sistema de Ecuaciones (Eliminación Gaussiana)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sistema de Ecuaciones (Gauss Jordan)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sistema de Ecuaciones (Sarrus)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Matrices</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Encriptar</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Desencriptar</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Entrada</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QTextEdit" name="textEdit_input">
             <property name="enabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QTextBrowser" name="textBrowser_result"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Resultaddo</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_ma">
          <property name="title">
           <string>Matriz A</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QTextEdit" name="textEdit_ma"/>
           </item>
           <item row="1" column="0">
            <widget class="QFrame" name="frame_encryption">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="1" column="0">
               <widget class="QLineEdit" name="lineEdit_e_message"/>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label_e_message">
                <property name="text">
                 <string>Mensaje</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QComboBox" name="comboBox_matrix_a_op">
             <item>
              <property name="text">
               <string>--Select--</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Traspuesta</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Inversa</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Determinante</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Rango</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="0" rowspan="2" colspan="3">
         <widget class="QSpinBox" name="spinBox_matrices_numeromatrices">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>2</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QGroupBox" name="frame_matrices_matrizb">
          <property name="title">
           <string>Matriz B</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="1">
            <widget class="QTextEdit" name="textEdit_mb"/>
           </item>
           <item row="0" column="0">
            <widget class="QComboBox" name="comboBox_matrix_ab_op">
             <item>
              <property name="text">
               <string>--Select--</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>*</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>+</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>-</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Markov</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="3" rowspan="3">
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>Resultado</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0">
            <widget class="QTextBrowser" name="textBrowser_matrix_result"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1035</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionsdfsadf">
   <property name="text">
    <string>sdfsadf</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
